junit-jupiter-params/src/main/java/org/junit/jupiter/params/converter/ArgumentConverter.java

```diff
@@ -40,7 +40,6 @@
  * the {@link ParameterContext} to perform the conversion.
  *
  * @since 5.0
- * @see SimpleArgumentConverter
```

Comment:
This was a duplicate:

https://github.com/junit-team/junit5/blob/529da15aa840ba4d063af94db915b7e81d5fa8d4/junit-jupiter-params/src/main/java/org/junit/jupiter/params/converter/ArgumentConverter.java#L43-L47